ERROR - 2016-07-21 09:04:41 - exception 'Twig_Error_Syntax' with message 'Unexpected "}" in "pages/myAccount.tpl" at line 19' in C:\xampp\htdocs\KinguinInternship\myProject\Twig\Lexer.php:259
Stack trace:
#0 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Lexer.php(216): Twig_Lexer->lexExpression()
#1 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Lexer.php(115): Twig_Lexer->lexVar()
#2 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Environment.php(469): Twig_Lexer->tokenize('{% include "./c...', 'pages/myAccount...')
#3 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Environment.php(559): Twig_Environment->tokenize('{% include "./c...', 'pages/myAccount...')
#4 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Environment.php(334): Twig_Environment->compileSource('{% include "./c...', 'pages/myAccount...')
#5 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Views\Twig.php(87): Twig_Environment->loadTemplate('pages/myAccount...')
#6 C:\xampp\htdocs\KinguinInternship\myProject\Yee\View.php(255): Yee\Views\Twig->render('pages/myAccount...', NULL)
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\View.php(243): Yee\View->fetch('pages/myAccount...', NULL)
#8 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(765): Yee\View->display('pages/myAccount...')
#9 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\MyAccount\MyAccountController.php(33): Yee\Yee->render('pages/myAccount...', Array)
#10 [internal function]: MyAccountController->indexAction()
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#12 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#13 [internal function]: MyAccountController::___indexAction()
#14 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('MyAccountContro...', Array)
#15 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#16 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#17 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#18 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#19 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#20 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#21 {main}
ERROR - 2016-07-21 09:05:41 - exception 'Twig_Error_Syntax' with message 'Unexpected "}" in "pages/myAccount.tpl" at line 19' in C:\xampp\htdocs\KinguinInternship\myProject\Twig\Lexer.php:259
Stack trace:
#0 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Lexer.php(216): Twig_Lexer->lexExpression()
#1 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Lexer.php(115): Twig_Lexer->lexVar()
#2 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Environment.php(469): Twig_Lexer->tokenize('{% include "./c...', 'pages/myAccount...')
#3 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Environment.php(559): Twig_Environment->tokenize('{% include "./c...', 'pages/myAccount...')
#4 C:\xampp\htdocs\KinguinInternship\myProject\Twig\Environment.php(334): Twig_Environment->compileSource('{% include "./c...', 'pages/myAccount...')
#5 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Views\Twig.php(87): Twig_Environment->loadTemplate('pages/myAccount...')
#6 C:\xampp\htdocs\KinguinInternship\myProject\Yee\View.php(255): Yee\Views\Twig->render('pages/myAccount...', NULL)
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\View.php(243): Yee\View->fetch('pages/myAccount...', NULL)
#8 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(765): Yee\View->display('pages/myAccount...')
#9 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\MyAccount\MyAccountController.php(34): Yee\Yee->render('pages/myAccount...', Array)
#10 [internal function]: MyAccountController->indexAction()
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#12 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___indexAction', Array)
#13 [internal function]: MyAccountController::___indexAction()
#14 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('MyAccountContro...', Array)
#15 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#16 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#17 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#18 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#19 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#20 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#21 {main}
ERROR - 2016-07-21 10:12:46 - exception 'ErrorException' with message 'Problem preparing query (UPDATE users SET `firstName` = ?, `lastName` = ?, `password` = ?) Unknown column 'firstName' in 'field list'' in C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\xampp\\htdocs...', 909, Array)
#1 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(350): Yee\Libraries\Database\MysqliDB->_buildQuery(NULL, Array)
#4 C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php(43): Yee\Libraries\Database\MysqliDB->update('users', Array)
#5 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\Ajax\AjaxController.php(105): App\Models\MyAccount\MyAccountModel->updateDetails()
#6 [internal function]: AjaxController->postUpdateMyAccount()
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#8 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___postUpdateMy...', Array)
#9 [internal function]: AjaxController::___postUpdateMyAccount()
#10 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#12 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#13 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#14 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#15 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#16 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#17 {main}
ERROR - 2016-07-21 10:14:19 - exception 'ErrorException' with message 'Undefined index: defalt' in C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php:41
Stack trace:
#0 C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php(41): Yee\Yee::handleErrors(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 41, Array)
#1 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\Ajax\AjaxController.php(105): App\Models\MyAccount\MyAccountModel->updateDetails()
#2 [internal function]: AjaxController->postUpdateMyAccount()
#3 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#4 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___postUpdateMy...', Array)
#5 [internal function]: AjaxController::___postUpdateMyAccount()
#6 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#8 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#9 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#10 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#12 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#13 {main}
ERROR - 2016-07-21 10:14:35 - exception 'ErrorException' with message 'Undefined variable: email' in C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php:41
Stack trace:
#0 C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php(41): Yee\Yee::handleErrors(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 41, Array)
#1 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\Ajax\AjaxController.php(105): App\Models\MyAccount\MyAccountModel->updateDetails()
#2 [internal function]: AjaxController->postUpdateMyAccount()
#3 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#4 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___postUpdateMy...', Array)
#5 [internal function]: AjaxController::___postUpdateMyAccount()
#6 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#8 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#9 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#10 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#12 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#13 {main}
ERROR - 2016-07-21 10:14:47 - exception 'ErrorException' with message 'Problem preparing query (UPDATE users SET `firstName` = ?, `lastName` = ?, `password` = ?) Unknown column 'firstName' in 'field list'' in C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\xampp\\htdocs...', 909, Array)
#1 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(350): Yee\Libraries\Database\MysqliDB->_buildQuery(NULL, Array)
#4 C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php(43): Yee\Libraries\Database\MysqliDB->update('users', Array)
#5 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\Ajax\AjaxController.php(105): App\Models\MyAccount\MyAccountModel->updateDetails()
#6 [internal function]: AjaxController->postUpdateMyAccount()
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#8 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___postUpdateMy...', Array)
#9 [internal function]: AjaxController::___postUpdateMyAccount()
#10 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#12 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#13 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#14 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#15 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#16 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#17 {main}
ERROR - 2016-07-21 10:16:46 - exception 'ErrorException' with message 'Problem preparing query (UPDATE users SET `firstName` = ?, `lastName` = ?, `password` = ? WHERE   email) Unknown column 'firstName' in 'field list'' in C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\xampp\\htdocs...', 909, Array)
#1 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(350): Yee\Libraries\Database\MysqliDB->_buildQuery(NULL, Array)
#4 C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php(43): Yee\Libraries\Database\MysqliDB->update('users', Array)
#5 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\Ajax\AjaxController.php(105): App\Models\MyAccount\MyAccountModel->updateDetails()
#6 [internal function]: AjaxController->postUpdateMyAccount()
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#8 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___postUpdateMy...', Array)
#9 [internal function]: AjaxController::___postUpdateMyAccount()
#10 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#12 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#13 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#14 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#15 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#16 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#17 {main}
ERROR - 2016-07-21 10:16:54 - exception 'ErrorException' with message 'Problem preparing query (UPDATE users SET `firstName` = ?, `lastName` = ?, `password` = ? WHERE   email) Unknown column 'firstName' in 'field list'' in C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\xampp\\htdocs...', 909, Array)
#1 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(350): Yee\Libraries\Database\MysqliDB->_buildQuery(NULL, Array)
#4 C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php(43): Yee\Libraries\Database\MysqliDB->update('users', Array)
#5 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\Ajax\AjaxController.php(105): App\Models\MyAccount\MyAccountModel->updateDetails()
#6 [internal function]: AjaxController->postUpdateMyAccount()
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#8 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___postUpdateMy...', Array)
#9 [internal function]: AjaxController::___postUpdateMyAccount()
#10 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#12 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#13 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#14 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#15 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#16 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#17 {main}
ERROR - 2016-07-21 10:17:18 - exception 'ErrorException' with message 'Problem preparing query (UPDATE users SET `firstName` = ?, `lastName` = ?, `password` = ? WHERE   email = ? ) Unknown column 'firstName' in 'field list'' in C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php:909
Stack trace:
#0 [internal function]: Yee\Yee::handleErrors(256, 'Problem prepari...', 'C:\\xampp\\htdocs...', 909, Array)
#1 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(909): trigger_error('Problem prepari...', 256)
#2 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(648): Yee\Libraries\Database\MysqliDB->_prepareQuery()
#3 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Libraries\Database\MysqliDB.php(350): Yee\Libraries\Database\MysqliDB->_buildQuery(NULL, Array)
#4 C:\xampp\htdocs\KinguinInternship\myProject\App\Models\MyAccount\MyAccountModel.php(43): Yee\Libraries\Database\MysqliDB->update('users', Array)
#5 C:\xampp\htdocs\KinguinInternship\myProject\App\Controllers\Ajax\AjaxController.php(105): App\Models\MyAccount\MyAccountModel->updateDetails()
#6 [internal function]: AjaxController->postUpdateMyAccount()
#7 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Managers\Controller\Controller.php(22): call_user_func_array(Array, Array)
#8 [internal function]: Yee\Managers\Controller\Controller::__callStatic('___postUpdateMy...', Array)
#9 [internal function]: AjaxController::___postUpdateMyAccount()
#10 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Route.php(462): call_user_func_array('AjaxController:...', Array)
#11 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1334): Yee\Route->dispatch()
#12 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\Flash.php(85): Yee\Yee->call()
#13 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\MethodOverride.php(92): Yee\Middleware\Flash->call()
#14 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Middleware\PrettyExceptions.php(67): Yee\Middleware\MethodOverride->call()
#15 C:\xampp\htdocs\KinguinInternship\myProject\Yee\Yee.php(1279): Yee\Middleware\PrettyExceptions->call()
#16 C:\xampp\htdocs\KinguinInternship\myProject\index.php(49): Yee\Yee->execute()
#17 {main}
